### DISCUSSION TITLE
Can Verify.DocNet ignore parts of a PDF page?

### DISCUSSION THREAD

**Discussion by kepar** [VerifyTests/Verify#667] [Q&A] [answered]
Hi,

I'm using Verify.DocNet to test the rendering of PDF-files.
I was wondering if there is any way to ignore certain parts of the resulting file?
maybe add a mask or something?
or maybe write and use my own comparer?

**Comment by SimonCropp** [comment, marked as answer]
currently no. that functionality does not exist. 

yes you can write your own comparer https://github.com/VerifyTests/Verify/blob/main/docs/comparer.md

it could also be implemented as a converter https://github.com/VerifyTests/Verify/blob/main/docs/converter.md
